core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -347,6 +347,19 @@ default boolean isComplete() {
      */
     CompletableFuture<Void> whenComplete();
 
+    /**
+     * Drain all data using {@link NoopSubscriber}.
+     *
+     * <p>For example:<pre>{@code
+     * CompletableFuture<Void> cf = StreamMessage.aborted(new RuntimeException()).subscribe();
+     * assert cf.isCompletedExceptionally();
+     * }</pre>
+     */
+    default CompletableFuture<Void> subscribe() {
+        subscribe(new NoopSubscriber<>());
```

Comment:
- Should we use `NoopSubscriber.get()`?
- We need to release / close the subscribed objects. It is not related to this PR. But there was a bug.
  Could we close the objects passed in `onNext(T t)`?
  You might want to check out `StreamMessageUtil.closeOrAbort()`.

Reply:
- I used `NoobSubscriber.get()` and it seems to me that there is no problem  What do you think about that? 

- As per your advice 🙇 ❤️ , I tried to solve the problem by adding `StreamMessageUtil.closeOrAbort()` to `NoopSubscriber#onNext`.